File: zOctober4th/ex33.c
/* André Cavalcante de Lima
33. Armazenar dez números na memória do computador.
Exibir os valores na ordem inversa à da digitação.

*/
int main(int argc, char const *argv[]){
    int numeros[10];

    printf("Digite 10 numeros ai: \n ");
    for(int i=0; i<=10; i++){
        scanf("%i", &numeros[i]);

    }
    printf("\nOs numeros digitados foram : \n\n");
    for(int i=10; i>=0; i--){

        printf("%i \n", numeros[i]);

    }
    return 0;
}

File: November8thPython 33-38/ex33.py
''' André Cavalcante de Lima
33. Armazenar dez números na memória do computador.
Exibir os valores na ordem inversa à da digitação.

'''

numeros=[]

for i in range(0,10,1):
    x = int(input("Digite 10 numeros ai: \n "))
    numeros.append(x)

    
print("\nOs numeros digitados foram : \n\n")
for i in range(9, -1, -1):
    
    print(numeros[i])

